src/Slab/CMakeLists.txt

```diff
@@ -33,7 +38,7 @@ if(NOT UNIX)  # Need to reinclude it on Windows
 endif()
 get_target_property(F_INTERFACE_COMPILE_FLAGS fortran_project_options INTERFACE_COMPILE_OPTIONS)
 message(DEBUG "Slab: F_INTERFACE_COMPILE_FLAGS=${F_INTERFACE_COMPILE_FLAGS}")
-target_link_libraries(Slab PRIVATE fortran_project_options)
+target_link_libraries(Slab PRIVATE SlabOptions fortran_project_options)
 
 install(TARGETS Slab DESTINATION PreProcess/GrndTempCalc)
 if(NOT ${STATIC_EXE})
```

Comment:
@jmarrec CMakeLists.txt for ...\src\Basement shows this as if(NOT ${FORTRAN_STATIC_EXE}). Should that key word also be used here?

Reply:
```suggestion
if(NOT ${FORTRAN_STATIC_EXE})
```

Reply:
Good catch, a revert conflict likely around https://github.com/NREL/EnergyPlus/commit/26b0c38fb6f09f5540946e5508cb3a1e1ee33bc3 . I made the replacement